package.json

```diff
@@ -67,14 +64,7 @@
     "chalk": "^2.4.2",
     "enquirer": "^2.3.4",
     "eslint": "^6.1.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-config-react-app": "^5.0.2",
-    "eslint-plugin-flowtype": "^3.13.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-react-hooks": "^2.2.0",
```

Comment:
I mentioned this in https://github.com/formium/tsdx/issues/634#issuecomment-674997895, but putting plugins and parsers as dependencies of a shareable config is explicitly not sanctioned by ESLint and against ESLint conventions, docs, etc. Other much more popular configs don't go against this convention either.